src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -29,65 +29,132 @@
+        </p>
+    </div>
+
+    <div class="column-two-thirds">
         <header class="page-header">
-            <table>
-                <tbody>
-                @titleCols.map { case (col1, col2) =>
-                <tr>
-                    <td>@Html(sectionLink(col1._1, col1._2))</td>
-                    <td>@col2.map { t => @Html(sectionLink(t._1, t._2)) }.getOrElse("")
-                </td>
-                </tr>
-                }
-                </tbody>
-            </table>
+            <h1 class="heading-xlarge">@opportunity.title</h1>
         </header>
 
+        <aside>
+            <nav role="navigation" class="contain-floats page-navigation">
+                <ol>
+                    @titleCols.map { case (col1, col2) =>
+                        <li>
+                            @Html(sectionLink(col1._1, col1._2))
+                        </li>
+                    }
+                </ol>
+                <ol>
+                    @titleCols.map { case (col1, col2) =>
+                        <li>
+                            @col2.map { t => @Html(sectionLink(t._1, t._2)) }.getOrElse("")
+                        </li>
+                    }
+                </ol>
+            </nav>
+        </aside>
+
+
         <div>
-            <h2>Apply for funding</h2>
-            <a href="@controllers.routes.ApplicationController.showOrCreateForForm(applicationId)">Make an application</a>
-            <p></p>
-            <a href="@controllers.routes.OpportunityController.showGuidancePage(opportunity.id)">Guidance on seminars</a>
-            <p></p>
+            <h2 class="heading-large">Apply for funding</h2>
+
+            <p>
+                <a href="@controllers.routes.ApplicationController.showOrCreateForForm(applicationId)">Make an
+                    application</a>
+            </p>
+
+            <p>
+                <a href="@controllers.routes.OpportunityController.showGuidancePage(opportunity.id)">Guidance on
+                    seminars</a>
+            </p>
         </div>
 
         @currentSection.map { s =>
-            <article>
-                <h1>@s.sectionNumber. @s.title</h1>
-                @if(sectionNumber == 1) {
-                    <div>
-                        <h3>Start date</h3>
-                        @opportunity.startDate
-                        <h3>Days duration</h3>
-                        @opportunity.duration.map(d => s"${d.duration} ${d.units}").getOrElse("Open indefinitely")
-                        <h3>Opportunity value</h3>
-                            &pound; @opportunity.value.amount @opportunity.value.unit
-                    </div>
-                    <p></p>
-                }
-                @s.paragraphs.map { p =>
-                    <p>@p</p>
-                }
-            </article>
-            <footer>
+
+            @if(sectionNumber == 1) {
+
+        <div class="panel panel-border-narrow">
```

Comment:
Well noticed.

I spoke to David on this.  I left this as-is for now to demonstrate the standard design pattern, but we agreed yesterday that it failed in this instance and the axshare version should be implemented.  

I see no problem in checking this in for now and reviewing after further consideration.